tools/telemetry/www/doc/TOC.md

```diff
@@ -0,0 +1,29 @@
+[HTML5 Boilerplate homepage](https://html5boilerplate.com)
```

Comment:
Though it does provide some guidance for future developers, but it might not be that necessary.